modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -34,6 +35,119 @@ admin_manual:release_notes.adoc
+
+Since version 10.5, xref:file-locking-in-the-web-interface[ownCloud supports Manual File Locking] to prevent concurrent changes of files by different users. The feature works in a way that it only allows the users who locked a file to unlock it again (_exclusive file locking_) and it provides a configurable timeout for file locks.
+
+Version 10.9 introduces a new sub-feature that allows administrators to define groups of users that are entitled to unlock locked files, independent of whether they have set the lock initially or not (https://github.com/owncloud/core/pull/38222[#38222]). Users in such groups can regain control over files, e.g., if a lock owner is unavailable and a long timeout for file locks is used. Administrators can find the configuration in the _Manual File Locking_ section of the _General_ admin settings.
+
+=== File Versions: Store and Display the Author of File Versions
+
+To improve usability, Server 10.9 brings a new feature for file versioning. When working on documents with multiple people it can be of interest to know who created a certain version, e.g., to roll back to a certain state or to understand who made which changes to a file. For this, you can optionally enable storing and displaying the author of file versions. When enabled, users will see the author of file versions in the 'Versions' tab of the right sidebar in the web interface. To avoid time-consuming database migrations, the author metadata will be stored in files on the storage once the feature has been enabled.
+
+The feature is disabled by default as it is not compatible with all use cases. To enable it, administrators can set `'file_storage.save_version_author' => true` in `config.php`.
+
+TIP: This feature is incompatible with S3 Object storage. Do not enable it if you're using S3 as a primary storage.
+
+=== Restrict Public Link Sharing to Certain Groups
+
+In certain scenarios it is not desired that all users of an ownCloud installation can create public links and share data publicly, respectively. To cover for this, ownCloud Server 10.9 comes with a new configuration option for administrators that allows restricting the creation of public links to users in certain groups (https://github.com/owncloud/core/pull/38980[#38980]).
+
+TIP: Existing public links will continue to work after introducing a restriction policy. The policy only applies to the creation of new public links.
+
+=== Ability to move a user's home folder
+
+In some cases it is necessary to manage storage locations of users, e.g., if you want to distribute them across multiple storages. To facilitate this process, Server 10.9 introduces a new _occ_ command (`occ user:move-home`) that allows moving user home folders to a new location (https://github.com/owncloud/core/pull/39136[#39136]). The command takes care of moving the physical data and takes care of updating all other related information.
+Administrators can find more information in the LINK[ownCloud documentation].
```

Comment:
FYI, the feature is documented at:
https://doc.owncloud.com/server/10.9/admin_manual/configuration/user/user_management.html
https://doc.owncloud.com/server/10.9/admin_manual/configuration/server/occ_command.html#user-commands

Note the related new `user:home` commands which are described at the same location